cpp/ps0000/ps0000_sol.cpp

```diff
@@ -1,9 +1,10 @@
-#include <iostream>
+#include <iostream> // 2/2
 
 using namespace std;
 
-int main()
+int main() // 1/1
 {
-    cout << "Hello, world!" << endl;
-    return 0;
+    cout << "Hello, World!" << endl; // 3/4

```

Comment:
- [x] used `<<` for outputting instead of `>>`
- [x] used `\n` or `std::endl` in the right way
- [x] avoided syntax error by typing `;` at the end of the line
- [ ] output is as required `Hello, World!` in a single line